File: java/get_opentypeextension_1.java
HttpResponse<String> response = Unirest.get("https://graph.microsoft.com/v1.0/me/messages(%27AAMkAGE1M2IyNGNmLTI5MTktNDUyZi1iOTVl===%27)/extensions(%27Com.Contoso.Referral%27)")
  .asString();

File: python/get_opentypeextension_1.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

conn.request("GET", "/v1.0/me/messages(%27AAMkAGE1M2IyNGNmLTI5MTktNDUyZi1iOTVl===%27)/extensions(%27Com.Contoso.Referral%27)")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
